src/components/ProfileHeader/ProfileHeader.test.tsx

```diff
@@ -0,0 +1,103 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import { afterEach } from "vitest";
+
+import {
+  ProfileHeader,
+  ProfileHeaderLogo,
+  ProfileHeaderMenuItem,
+  ProfileHeaderPopupMenuItem,
+  Icon
+} from '../';
+
+afterEach(() => {
```

Comment:
The global test setup file already has this `afterEach` cleanup block.  This one can be removed.